### PR TITLE
ncdf4 - netcdf4 for R 3.1.1 intel2014b

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a59a6c2918598eebb683 for a full test report.

**Comment by boegel** [comment]
Missing dependency on `cURL`?
```
ld: cannot find -lcurl
```

File: easybuild/easyconfigs/n/ncdf4/ncdf4-1.13-intel-2014b-R-3.1.1.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+]
+sources = ['%(name)s_%(version)s.tar.gz']
+
+r = 'R'
+rver = '3.1.1'
+versionsuffix = '-%s-%s' % (r, rver)
+
+dependencies = [
+    (r, rver),
+    ('netCDF', '4.3.2')
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncdf4/ncdf4-1.13-intel-2014b-R-3.1.1.eb]: syntax error: missing trailing `,`

**Comment by hajgato** [comment]
@boegel: commas corrected

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/bb10906e3072ac495262 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2742541fecde0d688214 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**